lib/widgets/fields/alignment_field.dart

```diff
@@ -0,0 +1,155 @@
+
+import 'package:flutter/material.dart';
+
+class AlignmentField extends StatefulWidget {
+  final void Function(Alignment) onChanged;
+
+  const AlignmentField({
+    Key? key,
+    required this.onChanged,
+  }) : super(key: key);
+
+  @override
+  _AlignmentFieldState createState() => _AlignmentFieldState();
+}
+
+class _AlignmentFieldState extends State<AlignmentField> {
+  int selectedOption = 5;
+  int onHoverOption = 5;
```

Comment:
give more meaningful name to variable according to its purpose i.e selectionOptionIndex , hoverOptionIndex.